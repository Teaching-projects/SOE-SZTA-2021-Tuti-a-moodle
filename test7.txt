-100
50
20
80
70
20