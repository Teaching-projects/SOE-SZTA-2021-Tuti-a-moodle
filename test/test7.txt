A
AL
-100
50
20
1
B
BL
80
70
20
1
