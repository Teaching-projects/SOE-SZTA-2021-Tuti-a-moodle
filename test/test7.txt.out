Exception in thread "main" java.lang.IllegalArgumentException: Health must not be negative
	at Entity.<init>(Entity.java:22)
	at Main.readEntity(Main.java:44)
	at Main.main(Main.java:14)
Enter the name of the entity: Enter the lore of A: Enter the health of A: Enter the attack of A: Enter the defense of A: Enter the cooldown of A: 